src/Uno.UWP/UI/Input/RightTappedEventArgs.cs

```diff
@@ -9,16 +9,19 @@ namespace Windows.UI.Input
 {
 	public partial class RightTappedEventArgs
 	{
-		internal RightTappedEventArgs(PointerDeviceType type, Point position)
+		internal RightTappedEventArgs(uint pointerId, PointerDeviceType type, Point position)
```

Comment:
Personal preference: usually I try to keep the properties -and their initialization- in the same order of the constructor parameters, I find it easier to read/maintain :)

Reply:
Agreed!